.github/workflows/trigger.yml

```diff
@@ -26,7 +26,7 @@ on:
 
 jobs:
   trigger:
-    runs-on: ubuntu-latest
+    runs-on: k8s-runner-e2e
```

Comment:
this workflow does not need to run on gke runner.
Please revert this change.

Reply:
reverted.